hedera-node/hedera-app/src/main/java/com/hedera/node/app/tss/handlers/TssSubmissions.java

```diff
@@ -154,6 +169,10 @@ private CompletableFuture<Void> submit(
                             body = builder.build();
                             try {
                                 appContext.gossip().submit(body);
+                                if (body.hasTssEncryptionKey()) {
```

Comment:
It will be easy to maintain it in `TssBaseServiceImpl`

Reply:
Removed this.